### PR TITLE
Add zn-CN translation for all nodes

### PR DESCRIPTION
**Pull request by JiyeYu**
## Description

I have added zn-CN translation for all nodes: Common, Function, Storage; network, parsers, sequence as well.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [x] For non-bugfix PRs, I have discussed this change on the mailing list/slack team.
- [x] I have run `grunt` to verify the unit tests pass
- [x] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: packages/node_modules/@node-red/nodes/locales/zh-CN/common/21-debug.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+-->
+
+<script type="text/x-red" data-help-name="debug">
+    <p>在“调试”侧边栏选项卡和运行时日志中显示选定的消息属性。 默认情况下，它会显示<code> msg.payload </code>，但也可以设置成显示任意属性，完整消息或JSONata表达式的结果。</p>
+    <h3>详细</h3>
+    <p>调试侧边栏会提供已发消息的结构化视图，方便您查询消息的结构。</p>
+    <p>JavaScript对象和数组可以根据需要来折叠或扩展。缓冲区对象可以显示为原始数据，也可以显示为字符串。</p>
+    <p>对任意条消息，调试侧边栏还会显示接收消息的时间，发送消息的节点以及消息类型等信息。单击源节点ID将在工作区中显示该节点。</p>
+    <p>节点上的按钮可用于启用或禁用其输出。建议禁用或删除所有未使用的调试节点。</p>
+    <p>还可以通过配置节点，将所有消息发送到运行时日志，或将简短的数据（32个字符内）在调试节点下的状态文本上显示。</p>
```
Comment (xiuqi08) [inline packages/node_modules/@node-red/nodes/locales/zh-CN/common/21-debug.html]: 将所有消息发送到运行时日志 -> 将所有消息发送到运行时的日志

File: packages/node_modules/@node-red/nodes/locales/zh-CN/function/90-exec.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    <p>（可选）可以选择使用<code>spawn</code>代替，它会在命令运行时从stdout和stderr返回输出，通常一次一行。完成后，它将在第三个端口上返回一个对象。例如，成功的命令应返回<code>{code：0}</code>。</p>
+    <p>错误可能会在第三个端口<code>msg.payload</code>上返回额外的信息，例如<code>message</code>字符串，<code>signal</code>字符串。</p>
+    <p>运行的命令是在节点内定义的，带有附加<code>msg.payload</code>的选项和另外一组参数。</p>
+    <p>带空格的命令或参数应该用引号引起来：<code>“这是一个参数”</code></p>
+    <p>返回的<code>payload</code>通常是<i>字符串</i>类型，除非检测到非UTF8字符，在这种情况下，它会是<i>buffer</i>类型。</p>
+    <p>节点处于活动状态时，该节点的状态图标和PID将可见。对此更改可以通过<code>Status</code>节点读取。</p>
+    <h4>杀死进程</h4>
+    <p>发送<code>msg.kill</code>将杀死一个活动进程。<code>msg.kill</code>应该是包含要发送的信号类型的字符串，例如<code>SIGINT</code>，<code>SIGQUIT</code>或<code>SIGHUP</code>。如果设置为空字符串，则默认为<code>SIGTERM</code>。</p>
+    <p>如果节点有多个进程在运行，则还必须设置<code>msg.pid</code>并设置要杀死的PID的值。</p>
+    <p>如果<code>Timeout</code>字段提供了一个值，则如果在指定的秒数过去后进程尚未完成，则该进程将自动终止。</p>
```
Comment (xiuqi08) [inline packages/node_modules/@node-red/nodes/locales/zh-CN/function/90-exec.html]: Before: <p>如果<code>Timeout</code>
After: <p>如果<code>超时</code>

**Comment by JiyeYu** [comment]
Hi @knolleary @dceejay , this pull request is ready to merge.

**Comment by knolleary** [comment]
Please can we confirm this PR is ready to merge?

**Comment by xiuqi08** [comment]
> Please can we confirm this PR is ready to merge?

I've reviewed the changes and it is ready to merge.

**Comment by JiyeYu** [comment]
Hi @knolleary 
This pull request is ready to merge now.

**Comment by knolleary** [comment]
Thank you everyone!

**Pull request merged**